templates/module/helfi_platform_config/profile-block.html.twig

```diff
@@ -0,0 +1,45 @@
+{% if logged_in %}
+  {% set link_attributes = {
+    'class': [
+      'profile__logout-link',
+    ],
+  } %}
+
+  {% set dropdown_id = 'profile-dropdown' %}
+
+  {% embed "@hdbt/misc/nav-toggle-button.twig" with {
+    modifier_class: 'nav-toggle--profile',
+    controls: dropdown_id,
+    anchor_target: '#profile',
+    js_target: 'js-profile-button',
+    modifier_suffix: 'profile',
+
+    open_label: display_name,
+    open_screenreader: 'You are logged in as '|t ~ display_name ~ '. ' ~ 'Open profile information menu'|t({}, {'context': 'Profile information-menu open button text for screen readers'}),
```

Comment:
Generally I feel that when we form a sentence from static text and variables, the variables should be within the translation, so that people translating could change the position of the variable within the translation.

```twig
open_screenreader: 'You are logged in as @display_name. Open profile information menu'|t({'@display_name': display_name},  {'context': 'Profile information-menu open button text for screen readers'}),
```

Reply:
Hmm good point. I will see what I can do.

Reply:
This is fixed.